packages/gestalt/src/GroupAvatar.js

```diff
@@ -181,6 +181,7 @@ export default function GroupAvatar(props: Props): Node {
       height={avatarHeight}
       position="relative"
       dangerouslySetInlineStyle={{
+        // $FlowFixMe[incompatible-type]
```

Comment:
Would require runtime changes. Better would be to change it to:

```
boxShadow: outline ? `0 0 0 2px ${colorGray0}` : undefined,
```

But then we should filter out all keys set in `__style` which have a value of `undefined`. Can be done in a follow up.